testing/workflows/components/kfctl_go_test.jsonnet

```diff
@@ -213,6 +213,27 @@ local dagTemplates = [
     ),
     dependencies: ["checkout"],
   },
+  // Verify Kubeflow is deployed successfully.
+  {
+    template: buildTemplate(
+      "kfctl-is-ready",
+      [        
+        "pytest",
+        "kf_is_ready_test.py",
+        // I think -s mean stdout/stderr will print out to aid in debugging.
+        // Failures still appear to be captured and stored in the junit file.
+        "-s",
+        // Increase the log level so that info level log statements show up.
+        "--log-cli-level=info",
+        // Test timeout in seconds.
+        "--timeout=500",
```

Comment:
if it's waiting for the deployment to be ready, 500 seconds seems to be too constraint?

Reply:
Intersting. It doesn't seem to work like that so I guess I can remove it.
If we look at the runs the tests are definitely running longer than 5 minutes.
http://testing-argo.kubeflow.org/workflows/kubeflow-test-infra/kubeflow-postsubmit-kfctl-go-4485a65-1297-efc5?tab=workflow